apps/api/src/app/environments/environment.ts

```diff
@@ -30,9 +30,11 @@ export default {
     wsdlUrl:
       process.env.HEALTH_INSURANCE_XROAD_WSDLURL ??
       'https://test-huld.sjukra.is/islandrg?wsdl',
-    baseUrl: process.env.XROAD_BASE_PATH ?? 'http://localhost:8080',
+    baseUrl: process.env.XROAD_BASE_PATH_WITH_ENV ?? 'http://localhost:8080',
```

Comment:
Ah. Now that I think about it. This environment variable probably won't work for you, since it includes the ENV, which is not the SOAP endpoint, right?

Please add the XROAD_BASE_PATH to application-system-api. I'll get other teams to migrate to the `${XROAD_BASE_PATH}/${SERVICE_PATH}/rest-api` pattern.

Reply:
Done changed